cmd/build-constants.go

```diff
@@ -27,6 +27,8 @@ var (
 	// Go get development tag.
 	goGetTag = "DEVELOPMENT.GOGET"
 
+	GOROOT = build.Default.GOROOT
```

Comment:
In what case you want `GOROOT`, Note that `runtime.GOROOT()` is always available

Reply:
`GOROOT` will cover paths like `<home_dir>/go`.
Removed the `GOROOT` definition from `cmd/build-constants.go` and used `runtime.GOROOT()` instead.